docs/providers/aws/events/rabbitmq.md

```diff
@@ -54,9 +54,10 @@ functions:
           basicAuthArn: arn:aws:secretsmanager:us-east-1:01234567890:secret:MySecret
 ```
 
-## Specifying batch size
```

Comment:
let's also reference this field in `serverless.yml.md` docs

Reply:
Added in ec95f579a4b1ac179856adab3bc5d6eba75f2df5

Reply:
Not sure why the PR is not showing the additional commit .. it’s definitely on the correct branch https://github.com/rm-hull/serverless/commit/ec95f579a4b1ac179856adab3bc5d6eba75f2df5

Reply:
Interesting - maybe you could close this PR then and open a new one?